requirements.txt

```diff
@@ -10,77 +10,76 @@
 -e git+https://github.com/nedbat/django_coverage_plugin@e18f49669f48dd7b49ab6d4de07442026f1f1978#egg=django_coverage_plugin
 ansicolors==1.1.8
 arrow==0.12.1
-beautifulsoup4==4.6.0
-bleach==2.1.3
-boto3==1.7.75
+beautifulsoup4==4.6.3
+bleach==3.0.2
+boto3==1.9.33
 boto==2.49.0
-botocore==1.10.84         # via boto3, s3transfer
+botocore==1.12.33         # via boto3, s3transfer
 brotlipy==0.7.0
-certifi==2018.4.16        # via requests
+certifi==2018.10.15       # via requests
 cffi==1.11.5              # via brotlipy
 chardet==3.0.4            # via requests
 configparser==3.5.0       # via tini
 coverage==4.5.1
 defusedxml==0.5.0         # via python3-openid
 dj-database-url==0.5.0
-django-allauth==0.37.1
+django-allauth==0.38.0
 django-appconf==1.0.2
-django-bootstrap-pagination==1.6.4
+django-bootstrap-pagination==1.7.0
 django-cors-headers==2.4.0
-django-debug-toolbar==1.9.1
-django-extensions==2.1.0
-django-filter==2.0.0
+django-debug-toolbar==1.10.1
+django-extensions==2.1.3
+django-filter==1.1.0
```

Comment:
Did you try running the code with these updates?

When I tried running our automated testing on this locally, I got an error similar to the one referenced by this issue, indicating the downgrade of `django-filter` is a problem: https://github.com/philipn/django-rest-framework-filters/issues/226

Reply:
fixed, and running the tests pass.  Getting some depreciation warnings that I will take care of in the next week or so.

Reply:
(also, playing around wiht the site manually also does not reveal any issues)